lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp

```diff
@@ -532,9 +555,72 @@ TEST_F(SessionManagerHandlerTest, test_SessionCompleteTerminationContext) {
   EXPECT_EQ(session_map[IMSI2].size(), 0);
 }
 
+TEST_F(SessionManagerHandlerTest, test_IPv6PDUSessionEstablishmentContext) {
+  magma::SetSMSessionContext request;
+  set_sm_session_context(&request);
+  request.mutable_common_context()->set_ue_ipv6(IPv6_5);
+
+  grpc::ServerContext server_context;
+
+  // create expected request for report_create_session call
+  CreateSessionRequest expected_request;
+  expected_request.mutable_common_context()->CopyFrom(request.common_context());
+  expected_request.mutable_rat_specific_context()->CopyFrom(
+      request.rat_specific_context());
+
+  // Create session request towards policydb
+  EXPECT_CALL(
+      *reporter, report_create_session(CheckSendRequest(expected_request), _))
+      .Times(1);
+
+  CreateSessionResponse response;
+  set_session_manager->SetAmfSessionContext(
+      &server_context, &request,
+      [this](grpc::Status status, SmContextVoid Void) {});
+
+  EXPECT_EQ(request.mutable_common_context()->sid().id(), IMSI1);
+
+  // Run session creation in the EventBase loop
+  evb->loopOnce();
+  SessionConfig cfg;
+  cfg.common_context       = request.common_context();
+  cfg.rat_specific_context = request.rat_specific_context();
+  cfg.rat_specific_context.mutable_m5gsm_session_context()->set_ssc_mode(
+      SSC_MODE_3);
+
+  auto session_map = session_store->read_sessions({IMSI1});
+  uint32_t pdu_id  = request.mutable_rat_specific_context()
+                        ->mutable_m5gsm_session_context()
+                        ->pdu_session_id();
+  set_session_manager->send_create_session(session_map, IMSI1, cfg, pdu_id);
+
+  auto it = session_map.find(IMSI1);
+  EXPECT_FALSE(it == session_map.end());
+  EXPECT_EQ(session_map[IMSI1].size(), 1);
+
+  auto& session_temp = session_map[IMSI1][0];
+  EXPECT_EQ(session_temp->get_config().common_context.sid().id(), IMSI1);
+  EXPECT_EQ(session_temp->get_config().common_context.ue_ipv6(), IPv6_5);
+
+  std::string session_id = id_gen_.gen_session_id(IMSI1);
+  auto credits           = response.mutable_credits();
+  create_credit_update_response(IMSI1, session_id, 1, 1024, credits->Add());
+  bool success = session_enforcer->m5g_init_session_credit(
+      session_map, IMSI1, session_id, cfg);
+  EXPECT_TRUE(success);
+  bool init_status =
+      session_store->create_sessions(IMSI1, std::move(session_map[IMSI1]));
```

Comment:
Why do you need to store it? 
What process are you checking?

Reply:
Removed this code as you suggested, as it is not needed here